docs/readme.md

```diff
@@ -180,20 +180,20 @@ Rocky.js currently implements a subset of Pebble's C-Style API. This section of
         <td>Implemented</td>
         <td>Framebuffer APIs will not be part of Rocky.js</td>
     </tr>
-    <tr class="planned">
+    <tr class="implemented">
         <td>[Drawing Text](https://developer.pebble.com/docs/c/Graphics/Drawing_Text/)</td>
-        <td>Planned</td>
+        <td>Implemented</td>
         <td></td>
     </tr>
-    <tr class="planned">
+    <tr class="partial">
         <td>[Fonts](https://developer.pebble.com/docs/c/Graphics/Fonts/)</td>
-        <td>Planned</td>
+        <td>Partial</td>
```

Comment:
My hunch was it was complete, just with some additional apis for embedded fonts. I'd rather a Pebbler make that call though :)